cmd/notary/main_test.go

```diff
@@ -159,10 +159,6 @@ var exampleValidCommands = []string{
 	"key list",
 	"key rotate repo snapshot",
 	"key generate rsa",
-	"key backup tempfile.zip",
-	"key export e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855 backup.pem",
```

Comment:
I think it'd be good to bring back the `key import` and `key export` commands here.  This just tests for invalid number of arguments, if I remember correctly.